lib/luatest_server.py

```diff
@@ -0,0 +1,110 @@
+import glob
+import os
+import re
+import sys
+from subprocess import STDOUT
+from gevent.subprocess import Popen, PIPE
```

Comment:
Why do you import Popen() and constants from different modules (subprocess and gevent.subprocess)?